packages/frontend/src/pages/settings/preferences.vue

```diff
@@ -236,6 +236,14 @@ SPDX-License-Identifier: AGPL-3.0-only
 										</MkSwitch>
 									</MkPreferenceContainer>
 								</SearchMarker>
+
+								<SearchMarker :keywords="['collapse', 'sensitive', 'channel', 'fold']">
+									<MkPreferenceContainer k="collapseSensitiveChannel">
+										<MkSwitch v-model="collapseSensitiveChannel">
+											<template #label><SearchLabel>{{ i18n.ts.collapseSensitiveChannel }}</SearchLabel></template>
+										</MkSwitch>
```

Comment:
```suggestion
										<MkSwitch v-model="collapseSensitiveChannel">
											<template #label><SearchLabel>{{ i18n.ts.collapseSensitiveChannel }}</SearchLabel></template>
											<template #caption>{{ i18n.ts.collapseSensitiveChannelDescription }}</template>
										</MkSwitch>
```

- `i18n.ts.collapseSensitiveChannel`: センシティブチャンネルの投稿を隠す
- `i18n.ts.collapseSensitiveChannelDescription`: そのチャンネル内・通知・ホームタイムラインなど、一部の場所では引き続き表示されます。

とかのほうが良いかもしれない

Reply:
分けるのありですね。

しかしさやまめさんと話しして、ちょっと仕様をもう少し変えるかもです。タイムラインごとの上の設定にするかグローバルな設定にするべきかを少し悩んでます

Reply:
設定が増えすぎても大変＋センシティブなメディアを含むノートは今のところTL限定且つ一律なのでそこの挙動との一貫性を考えたほうが良いかも

Reply:
グローバルな設定を維持し、HTLの動作を改善することにしました。

設定項目については取り込みました in 28ea454b